index.bs

```diff
@@ -643,6 +643,19 @@ It has the following constraints:
 * Every element's [=report window/start=] is equal to the previous element's [=report window/end=], if it exists.
 * There is at least one element in the list.
 
+<h3 id="trigger-specs-header">Trigger specs</h3>
+
+A <dfn>trigger spec</dfn> is a [=struct=] with the following items:
+
+<dl dfn-for="trigger spec">
+: <dfn>event-level report windows</dfn>
+:: A [=report window list=].
+
+</dl>
+
+A <dfn>trigger spec map</dfn> is a [=map=] whose keys are unsigned 32-bit
+integers and values are [=trigger specs=].
```

Comment:
The values here seem to lead to the same header link in the webpage. Is this a recursive definition? I didn't understand trigger specs as a recursive structure.

Reply:
Oh, right. I thought the map was a second entry in the definition of trigger spec. Sorry, I need to improve my reading accuracy.